test/e2e/load.go

```diff
@@ -46,9 +48,9 @@ const (
 	smallRCSize       = 5
 	mediumRCSize      = 30
 	bigRCSize         = 250
-	smallRCGroupName  = "load-small-rc"
-	mediumRCGroupName = "load-medium-rc"
-	bigRCGroupName    = "load-big-rc"
+	smallRCGroupName  = "load-small"
```

Comment:
If you're touching it, you should probably rename it to "smallGroupName" (without RC)? Or sth like that.